cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ConditionArgument.java

```diff
@@ -289,7 +289,7 @@ public void leavingState(ParsingContext ctx) throws CommandFormatException {
         @Override
         public void character(ParsingContext ctx) throws CommandFormatException {
             //System.out.println(ctx.getState().getId() + ": " + ctx.getCharacter());
-            if(parenthesisState != ctx.getState() && !Character.isWhitespace(ctx.getCharacter())) {
+            if(parenthesisState != ctx.getState() && (!Character.isWhitespace(ctx.getCharacter()) || ctx.getState().getId().equals(QuotesState.ID) || ctx.getState().getId().equals(EscapeCharacterState.ID))) {
```

Comment:
@RanabirChakraborty just a minor comment, could you cut this line after the first '||'. Thank-you.

Reply:
Thanks for the review @jfdenise, Did the changes.